Sources/SwiftDocC/Semantics/Metadata/Availability.swift

```diff
@@ -92,11 +103,11 @@ extension Metadata {
 
         /// The platform version that this page was introduced in.
         @DirectiveArgumentWrapped
-        public var introduced: String
+        public var introduced: SemanticVersion
```

Comment:
We synced offline about this change and decided to go with changing the type now to avoid changing this property multiple times.